README.md

```diff
@@ -73,15 +69,11 @@ jobs:
 
       - uses: im-open/octostache-action@v1.0.0
         with:
-          variablesFile: ./substitution-variables.json
-          templateFile: ./src/DemoApp19/DemoApp19.csproj
-          outputFile: ./src/DemoApp19/DemoApp19.csproj
-
-      - uses: im-open/octostache-action@v1.0.0
-        with:
-          variablesFile: ./substitution-variables.json
-          templateFile: ./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
-          outputFile: ./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
+          variables-file: ./substitution-variables.json
+          files-with-substitutions: ./src/DemoApp19/DemoApp19.csproj,./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
+          output-files: ./src/DemoApp19/DemoApp19.csproj,./src/DemoApp19/Bff/FrontEnd/scripts/build-variables.js
```

Comment:
I'm curious about this and it's something I wondered about with the original action.  Is there a compelling reason to allow it to output to a different file?  Would it simplify things if we just said the action reads/writes to the same file?

@izep

Reply:
Makes testing easier for sure, I'll say that much. Other than that 🤷